script.js

```diff
@@ -0,0 +1,77 @@
+class BookLibrary {
+  constructor() {
+    this.myBooks = [];
+    this.addBtn = document.getElementById('add-btn');
+    this.listBooks = document.getElementById('list-books');
+
+    const storedBooks = localStorage.getItem('myBooks');
+    if (storedBooks) {
+      this.myBooks.push(...JSON.parse(storedBooks));
+    }
+
+    this.addBtn.addEventListener('click', this.addBook.bind(this));
+
+    // Initial display of books
+    this.showBooks();
+  }
+
+  showBooks() {
+    this.listBooks.innerHTML = '';
+
+    this.myBooks.forEach((myBook) => {
+      const myBookDiv = document.createElement('div');
+      myBookDiv.classList.add('row');
+
+      const bookDetails = document.createElement('h4');
+      bookDetails.textContent = `${myBook.bookTitle} by ${myBook.bookAuthor}`;
+
+      const removeBtn = document.createElement('button');
+      removeBtn.classList.add('remove-btn');
+      removeBtn.textContent = 'Remove';
+      removeBtn.addEventListener('click', () => {
+        this.removeBook(myBook.bookTitle);
+      });
+
+      myBookDiv.appendChild(bookDetails);
+      myBookDiv.appendChild(removeBtn);
+      this.listBooks.appendChild(myBookDiv);
+    });
+  }
+
+  addBook() {
+    const bookTitle = document.getElementById('title').value.trim();
+    const bookAuthor = document.getElementById('author').value.trim();
+    const errorMsg = 'Both Title and Author must be filled out!';
+
+    if (bookTitle === '' || bookAuthor === '') {
+      // eslint-disable-next-line no-alert
```

Comment:
- [ ] You have done perfect work, however, we have a few points of improvement, talking about linter, it helps us clean and standardize our code regarding best practices and DRY method, to say so, I believe it is not a good practice to disable it in our code, I would strongly recommend you fix it up and remove the disabling lines as this will contribute to your professionalism and gain some good ways of writing a clean code in the near future, what do you think about it? 😃